lib/Benchmark/Metadata/Driver/AttributeDriver.php

```diff
@@ -39,7 +40,7 @@ public function getMetadataForHierarchy(ReflectionHierarchy $hierarchy): Benchma
     private function buildBenchmark(BenchmarkMetadata $benchmark, ReflectionHierarchy $hierarchy): void
     {
         $attributes = [];
-        $reflectionHierarchy = array_reverse(iterator_to_array($hierarchy));
+        $reflectionHierarchy = iterator_to_array($hierarchy);
```

Comment:
This changes how subjects are scanned: child classes (based on my assumption of `ReflectionHierarchy` order) appear first. Parent class methods are skipped, if subjects were already generated for those.

Since attributes in reflection are inherited (big difference, compared to `doctrine/annotations`), we don't need to scan parents first.